python_src/src/lamp_py/postgres/postgres_schema.py

```diff
@@ -144,6 +144,7 @@ class StaticFeedInfo(SqlBase):  # pylint: disable=too-few-public-methods
     feed_start_date = sa.Column(sa.Integer, nullable=False)
     feed_end_date = sa.Column(sa.Integer, nullable=False)
     feed_version = sa.Column(sa.String(75), nullable=False, unique=True)
+    version_pub_date = sa.Column(sa.Integer, nullable=True, index=True)
```

Comment:
Open to name suggestions on this column. It's kind of a modified `feed_start_date`

Reply:
`feed_activation_date`? its the date that this schedule is going to start being used, right?

also, do we want it to be nullable? aren't we going to be using this column to decide which schedule we want to use?

Reply:
Incorporated, changed to `feed_active_date` and made non-nullable, along with a migration call to populate existing records in dev RDS.